dev/source/docs/ardupilot-on-linux-starting.rst

```diff
@@ -0,0 +1,58 @@
+.. _ardupilot-on-linux-starting:
+
+===============
+Startup Options
+===============
+
+Run a build with -\-help to see the latest command-line options.
+
+-\-serial#
```

Comment:
need to specify that the user running ardupilot should be dialout group to have control on serial and video for video stream. (prevent having people doing chmod 777 /dev/ttyAM0

Reply:
Got it. I had to run mine as root to be able to mmap(/dev/mem), so I hadn't noticed.